Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -86,7 +90,13 @@ def bump_formula_pr
 
     old_formula_version = formula_version(formula, requested_spec)
 
-    replacement_pairs = if new_url_hash
+    replacement_pairs = if requested_spec == :stable && formula.revision != 0
+      [[/^  revision \d+\n/m, ""]]
+    else
+      []
+    end
```

Comment:
I'm glad you noticed. I was secretly hoping to get this comment. 🙈 Of course which one is more appropriate completely depends on the context.